src/lib/dlt_client.c

```diff
@@ -199,7 +197,7 @@ DltReturnValue dlt_client_connect(DltClient *client, int verbose)
         freeaddrinfo(servinfo);
 
         if (p == NULL) {
-            dlt_log(LOG_ERR, "ERROR: failed to connect.\n");
+            dlt_vlog(LOG_ERR, "ERROR: failed to connect! %s\n", strerror(errno));
```

Comment:
This line should return errno from connect() system call. Means if some lines are added in future between the two lines, the errno might be updated and this log doesn't return expected value. To make is clear, how about introducing a new local variable e.g. `tmp_errno`, store errno into `tmp_errno` in line 190, and print the value here?

Reply:
I've added `connect_errno`. Let me know if this is ok.